### ISSUE TITLE
Add more options to index management -> indices screen

### ISSUE THREAD

**Issue by aditjind** [opensearch-project/index-management#54]
<a href="https://github.com/ict-one-nl"><img src="https://avatars.githubusercontent.com/u/2643715?v=4" align="left" width="96" height="96" hspace="10"></img></a> **Issue by [ict-one-nl](https://github.com/ict-one-nl)**
_Sunday Dec 27, 2020 at 12:14 GMT_
_Originally opened as https://github.com/opendistro-for-elasticsearch/index-management/issues/365_

----

**Is your feature request related to a problem? Please describe.**
Deleting an index, updating it's settings, etc. would be nice to have here like Kibana from ElasticSearch itself.

**Describe the solution you'd like**

**Describe alternatives you've considered**
REST API is fine but a graphical interface would be nice as well.

**Additional context**

**Comment by thalurur** [comment]
@dswitzer you can create a policy with a delete action and attach it any index to automate delete of indices based on conditions - https://docs-beta.opensearch.org/docs/im/ism/policies/#delete

**Comment by dswitzer** [comment]
@thalurur Thanks for the suggestion, but it's not really want I'm wanting. We've got a multi tenant application and occasionally we may need to manually perform actions (e.g. deleting an index) for an inactive tenant. This is not something we can trigger based on policies alone. The X Pack version of the _Index Management_ has support for the types of actions we need. I'd just love to see this rolled into the Open Distro plugin to avoid having to build the functionality manually.

**Comment by thalurur** [comment]
Apologies, but I am trying to understand a little better about what the ask is - I am assuming by manual action one would trigger DELETE action and it deletes the index, but then its same as DELETE API provided in the core library right? is there a difference between the two that I am not seeing?

**Comment by ZeevGerstner** [comment]
I would love this feature too, it's really missing and make it hard to migrate from es to open distro.
@dswitzer is referring to this section in es:
![Screenshot 2021-06-23 134631](https://user-images.githubusercontent.com/42205609/123084100-6fd66000-d429-11eb-98db-50422f2eb972.png)

**Comment by dswitzer** [comment]
Correct. The Lifecycle buttons have actions you can take on an index. You
can certainly do this via the API, but having a UI to do some simple
management cleanup is convenient, especially when you might have tasks that
you want to distribute to users who might not have access to the API or you
don’t want them too.
